### PR TITLE
Handle tracking preference as string in database

### PR DESCRIPTION
**Pull request by CPatchane**

### CODE REVIEW

**Comment by CPatchane** [comment]
[ETU l'option de tracking est prise en compte dans settings](https://trello.com/c/ATeWaBs9/488-etu-loption-de-tracking-est-prise-en-compte-dans-settings)

**Comment by CPatchane** [comment]
@gregorylegarec We could use a boolean in the database, it's just an exception for the tracking since it's serve by the stack on the [`.ejs` file](https://github.com/cozy/cozy-settings/blob/master/src/index.ejs#L28) for the tracking enabling.

File: src/actions/index.js [hunk 1]
```diff
@@ -36,6 +36,10 @@ export const fetchInfos = () => {
     dispatch({ type: FETCH_INFOS })
     cozyFetch('GET', '/settings/instance')
       .then(instance => {
+        // tracking preference is stored as string, convert it to boolean for the checkbox
+        if (instance && instance.data && instance.data.attributes && instance.data.attributes.hasOwnProperty('tracking')) {
```
Comment (gregorylegarec) [inline src/actions/index.js]: Why `hasOwnProperty` ?

**Comment by gregorylegarec** [comment]
@CPatchane Ok, so it looks like we are hacking a stack issue here. :D
One more question and we're done. :)

File: src/actions/index.js (hunk 1 above)
Comment (CPatchane) [inline src/actions/index.js]: Don't you think we should handle falsy value here? Like empty string?

Comment (gregorylegarec) [inline src/actions/index.js]: As we only interested in a string having a `true` value, I don't think so.

**Comment by gregorylegarec** [comment]
Thanks @CPatchane !

**Comment by gregorylegarec** [comment]
[(?) Injecter la variable de tracking aux applications](https://trello.com/c/aHcFqunq/529-injecter-la-variable-de-tracking-aux-applications)

**Pull request merged**